README.md

```diff
@@ -50,6 +50,27 @@ Migration filename: [<enter>=declare_schema_migration_1|<custom_name>]: add_comp
 ```
 Note that the migration generator is interactive -- it can't tell the difference between renaming something vs. adding one thing and removing another, so sometimes it will ask you to clarify.
 
+## Migrator Configuration
+
+The following configuration options are available for the gem and can be used
+during the initialization of your Rails application.
+
+### after_load_rails_models callback
+
+During the initializtion process for generating migrations, `DeclareSchema` will
+trigger the `eager_load!` on the `Rails` application and all `Rails::Engine`s loaded
+into scope.  This sometimes won't load all the necessary models for generating
+migrations against, so we've introduced a callback that when defined is executed just
+after this eager loading process.
+
+**Example Configuration**
+
+```ruby
+DeclareSchema::Migration::Migrator.after_load_rails_models do
```

Comment:
I wonder if it will be clear that this is just used for migrations? Let's try to come up with a name that captures that. And I'm not sure the order matters, so "after" might not be important. How about
```
DeclareSchema::Migration::Migrator.load_models_for_migration do
```
or maybe
```
DeclareSchema::Migration::Migrator.load_other_models_for_migration do
```
or maybe just
```
DeclareSchema::Migration::Migrator.models_for_migration do
```
?

Reply:
I was thinking that this hook only being for during migrations was evident by the context of the class it's attached to as well as the documentation. I'm good with renaming it, but I don't think the `for_migration` is necessary since it's attached to the migration generator.

Reply:
Just because it's attached to the Migrator doesn't make it clear that it's _only_ used when generating migrations.

Here's the comment in the code today:

>   Call Fact Shards are optimized to be loaded only when needed, load these when the declare_schema migrator is running.

So how about:
```
DeclareSchema::Migration::Migrator. before_generating_migration do
```

Reply:
That naming sounds good to me, I'll push that up now